### PR TITLE
Added missing user events properties and types

### PR DESCRIPTION
**Pull request by mkliche**
Added some missing properties listed on https://github.com/binance-exchange/binance-official-api-docs/blob/master/user-data-stream.md and also separated the "Message" type into OutboundAccountInfo and ExecutionReport

### CODE REVIEW

**Comment by balthazar** [comment]
@kaminskypavel @bennyn

File: types.d.ts [hunk 1]
```diff
@@ -184,4 +205,42 @@ declare module 'binance-api-node' {
...
+        buyerCommissionRate: number;
+        sellerCommissionRate: number;
+        canTrade: boolean;
+        canWithdraw: boolean;
+        canDeposit: boolean;
+        lastAccountUpdate: number;
+    }
+
+    interface ExecutionReport extends Message {
+        symbol: string;
```
Comment (bennyn) [inline types.d.ts]: Let's sort all these attributes alphabetically, otherwise it's hard to maintain them.

Comment (austinyearlykim) [reply types.d.ts]: ++

**Comment by balthazar** [comment]
If there is no other concern besides the alphabetical thing I'm going to merge this, and we can change it if we really want to. I found that having to deal with this ordering manually was usually a pain, maybe a prettier plugin will save us one day 🙂

**Comment by bennyn** [comment]
👍

**Pull request merged**